doc/classes/ProjectSettings.xml

```diff
@@ -5,6 +5,8 @@
 	</brief_description>
 	<description>
 		Contains global variables accessible from everywhere. Use [method get_setting], [method set_setting] or [method has_setting] to access them. Variables stored in [code]project.godot[/code] are also loaded into ProjectSettings, making this object very useful for reading custom game configuration options.
+
```

Comment:
There should not be empty lines in the XML as each will become a separate `<p>`, which is not necessary here.
Could you amend the commit and force push to fix it? Otherwise it seems good.

Reply:
Done (2a642e1). Sorry for the delay, busy day yesterday.